docs/content/users/install/docker-installation.md

```diff
@@ -22,7 +22,7 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     colima start --cpu 4 --memory 6 --disk 100 --vm-type=qemu --mount-type=sshfs --dns=1.1.1.1
     ```
     (On macOS versions before Ventura, omit the `--vm-type=qemu` flag as it doesn't work on older OS versions.)
-    4. After [installing DDEV](ddev-installation.md), configure your system to use Mutagen—essential for DDEV with Colima—with `ddev config global --mutagen-enabled`.
+    4. After [installing DDEV](ddev-installation.md), make sure Mutagen is not disabled — essential for DDEV with Colima — with `ddev config global --performance=default`.
```

Comment:
Can we possibly just omit this line now?

```suggestion
```

Reply:
I had it removed but restored again afterwards to give people which may have it turned of a hint how to restore. But I'm also fine removing it again.